pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -76,6 +77,7 @@ while [ "$#" -gt 0 ]; do
         ;;
       --use-substitutes|-s)
         copyClosureFlags+=("$i")
+        copyFlags+=("-s")
```

Comment:
Whats the difference between copyFlags and copyClosureFlags?

Reply:
Former is for `nix copy`, latter is for `nix-copy-closure`. I can't exactly reuse because `nix copy`'s flag is named differently. Technically I could move both to use the same array name and instead of using $i they will always use -s. What do you think? Both commands support -s as shorter for this functionality, the longer version for nix copy is --substitute-on-destination while for nix-copy-closure it's --use-substitutes.

Reply:
I updated it so all of them work the same.